### DISCUSSION TITLE
Why My Student Application got rejected and I cant able to apply again

### DISCUSSION THREAD

**Discussion by VDplayz** [community/community#122199] [GitHub Education]
### Hi everyone,

Hi I need Help, I applied for Student developer pack and it got rejected becuase of my school is not listed in github and when i try to verify it again It just take me to this site : https://github.com/pricing Please Github Help me

**Comment by nafargi** [comment]
I understand your frustration. It's unfortunate that your school isn't listed on GitHub for verification. Here are some steps you can take to resolve the issue:

1. **Contact GitHub Support**: You can reach out to GitHub Support directly and explain your situation. Provide them with any documentation or proof of your student status, such as a student ID card or enrollment certificate. They might be able to assist you in verifying your student status manually.

2. **Contact Your School**: Reach out to your school administration or IT department and inquire if they have any process in place to register the institution with GitHub for the Student Developer Pack. They may be able to provide guidance or initiate the registration process.

3. **Explore Alternative Verification Methods**: GitHub offers alternative verification methods for students who are unable to verify their status through their school email address. You can explore options such as using an academic email address issued by your school, an official enrollment letter, or a school-issued document with your name and enrollment status.

4. **Appeal the Rejection**: If your application for the Student Developer Pack was rejected, you may have the option to appeal the decision. Provide any additional information or documentation that supports your student status and explains why your application should be reconsidered.

Remember to remain patient and persistent in resolving the issue. GitHub's support team is there to assist you, and they'll do their best to help you verify your student status and access the benefits of the Student Developer Pack.

**Reply by VDplayz** [reply]
I tried to get help from github support but they don't help and I have student documents and my school is not listed in github and when i try to reapply it will take me to github pricing
